include/roaring/roaring.h

```diff
@@ -479,14 +479,22 @@ size_t roaring_bitmap_shrink_to_fit(roaring_bitmap_t *r);
  * more space efficient than the portable form, e.g. when the data is sparse.
  *
  * Returns how many bytes written, should be `roaring_bitmap_size_in_bytes(r)`.
+ *
+ * This function is endian-sensitive: big-endian systems will not be able toCROARING_
```

Comment:
Looks like a find/replace issue, seems to be in ~every changed doc comment below

Reply:
Damn it. Will fix.

Reply:
fixed.